turbopack/crates/turbopack-ecmascript/tests/tree-shaker/analyzer/route-handler/output.md

```diff
@@ -107,64 +107,55 @@ graph TD
 # Final
```

Comment:
Original:
<img width="1046" alt="image" src="https://github.com/user-attachments/assets/c3a09af7-7639-4144-8fdb-b8939c628024" />


New: 
<img width="1046" alt="image" src="https://github.com/user-attachments/assets/d07e0e5f-01a7-4c36-9740-152058272f37" />

This is kinda obvious